### PR TITLE
Added react router, created components: MovieCard, LogIn

### PR DESCRIPTION
**Pull request by ajackowska**

### CODE REVIEW

File: src/components/MovieCard.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from "react"
```
Comment (marcin-kwiatkowski) [inline src/components/MovieCard.js]: ./src/components/MovieCard.js
  Line 3:22:  'Router' is defined but never used  no-unused-vars
  Line 4:5:   'Route' is defined but never used   no-unused-vars

File: src/components/Header.js [hunk 2]
```diff
@@ -1,19 +1,27 @@
...
 
 
 function Header() {
     return (
         <div className="header">
             <div className="header-container">
                 <div className="header-logo">
-                    <img src={Logo} alt="Logo" height="75px"></img>
+                    <Link to="/">
+                        <img src={Logo} alt="Logo" height="75px"></img>
```
[the remark this answers is not in the document]
Comment (ajackowska) [reply src/components/Header.js]: Poprawione;)

File: src/components/MovieCard.js (hunk 1 above)
Comment (ajackowska) [reply src/components/MovieCard.js]: ok, rozumiem;)

File: src/App.js [hunk 3]
```diff
@@ -1,18 +1,31 @@
...
-      <SearchBar />
-      <MovieTile />    
-      <Footer />     
-  </div>
-}
+  return (
+    <Router>
+      <div className="container">
+        <Header />
+
```
[the remark this answers is not in the document]
Comment (ajackowska) [reply src/App.js]: poprawione;), będę tego pilnować

**Pull request merged**